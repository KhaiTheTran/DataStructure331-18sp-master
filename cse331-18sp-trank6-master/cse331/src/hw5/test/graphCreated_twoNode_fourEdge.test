# Test graphCreated_twoNode_fourEdge

# Create a graph
CreateGraph graphCreated_twoNode_fourEdge

# Add two nodes
AddNode graphCreated_twoNode_fourEdge xx
AddNode graphCreated_twoNode_fourEdge yy

# Add four edges
AddEdge graphCreated_twoNode_fourEdge xx xx xxxx
AddEdge graphCreated_twoNode_fourEdge xx yy xxyy
AddEdge graphCreated_twoNode_fourEdge yy xx yyxx
AddEdge graphCreated_twoNode_fourEdge yy yy yyyy

# Print the nodes in the graph and the outgoing edges from xx,
# and outgoing edge from yy
ListNodes graphCreated_twoNode_fourEdge
ListChildren graphCreated_twoNode_fourEdge xx
ListChildren graphCreated_twoNode_fourEdge yy