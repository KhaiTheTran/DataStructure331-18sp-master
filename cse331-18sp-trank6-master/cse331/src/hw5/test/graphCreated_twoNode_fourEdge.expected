# Test graphCreated_twoNode_fourEdge

# Create a graph
created graph graphCreated_twoNode_fourEdge

# Add two nodes
added node xx to graphCreated_twoNode_fourEdge
added node yy to graphCreated_twoNode_fourEdge

# Add four edges
added edge xxxx from xx to xx in graphCreated_twoNode_fourEdge
added edge xxyy from xx to yy in graphCreated_twoNode_fourEdge
added edge yyxx from yy to xx in graphCreated_twoNode_fourEdge
added edge yyyy from yy to yy in graphCreated_twoNode_fourEdge

# Print the nodes in the graph and the outgoing edges from xx,
# and outgoing edge from yy
graphCreated_twoNode_fourEdge contains: xx yy
the children of xx in graphCreated_twoNode_fourEdge are: xx(xxxx) yy(xxyy)
the children of yy in graphCreated_twoNode_fourEdge are: xx(yyxx) yy(yyyy)